tests/entities/repos/test_filing_repo.py

```diff
@@ -0,0 +1,244 @@
+from regtech_cleanup_api.entities.repos import submission_repo as sub_repo
+from sqlalchemy.orm import Session, scoped_session
+from datetime import datetime as dt
+
+
+class TestFilingRepo:
+    @pytest.fixture(scope="function", autouse=True)
+    def setup(self, transaction_session: Session, session_generator: scoped_session):
+        user_action1 = UserActionDAO(
+            id=1,
+            user_id="test@local.host",
+            user_name="signer name",
+            user_email="test@local.host",
+            action_type=UserActionType.SIGN,
+            timestamp=dt.now(),
+        )
+        user_action2 = UserActionDAO(
+            id=2,
+            user_id="test@local.host",
+            user_name="submitter name",
+            user_email="test@local.host",
+            action_type=UserActionType.SUBMIT,
+            timestamp=dt.now(),
+        )
+        user_action3 = UserActionDAO(
+            id=3,
+            user_id="test@local.host",
+            user_name="accepter name",
+            user_email="test@local.host",
+            action_type=UserActionType.ACCEPT,
+            timestamp=dt.now(),
+        )
+        user_action4 = UserActionDAO(
+            id=4,
+            user_id="test@local.host",
+            user_name="creator name",
+            user_email="test@local.host",
+            action_type=UserActionType.CREATE,
+            timestamp=dt.now(),
+        )
+
+        transaction_session.add(user_action1)
+        transaction_session.add(user_action2)
+        transaction_session.add(user_action3)
+        transaction_session.add(user_action4)
+
+        filing_period = FilingPeriodDAO(
+            code="2024",
+            description="Filing Period 2024",
+            start_period=dt.now(),
+            end_period=dt.now(),
+            due=dt.now(),
+            filing_type=FilingType.ANNUAL,
+        )
+        transaction_session.add(filing_period)
+
+        filing1 = FilingDAO(
+            id=1,
+            lei="1234567890",
+            institution_snapshot_id="Snapshot-1",
+            filing_period="2024",
+        )
+        filing2 = FilingDAO(
+            id=2,
+            lei="ABCDEFGHIJ",
+            institution_snapshot_id="Snapshot-1",
+            filing_period="2024",
+        )
+        filing3 = FilingDAO(
+            id=3,
+            lei="ZYXWVUTSRQP",
+            institution_snapshot_id="Snapshot-1",
+            filing_period="2024",
+        )
+        filing1.creator = user_action4
+        filing2.creator = user_action4
+        filing3.creator = user_action4
+
+        transaction_session.add(filing1)
+        transaction_session.add(filing2)
+        transaction_session.add(filing3)
+
+        submission1 = SubmissionDAO(
+            id=1,
+            filing=1,
+            submitter_id=2,
+            state=SubmissionState.SUBMISSION_UPLOADED,
+            validation_ruleset_version="v1",
+            submission_time=dt.now(),
+            filename="file1.csv",
```

Comment:
Submissions now have a field called "counter", so make sure you pull in the latest Filing API.  One of the issues with dependencies across repos.  So I believe these DAOs will need to be updated.

Reply:
Will pull the latest and make any changes related to it.